tests/test_execute.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Integration tests for the ``execute`` method of LightningQubit.
+"""
+import pytest
+
+import pennylane as qml
+from pennylane import numpy as np
+
+
+@pytest.mark.parametrize("diff_method", ("param_shift", "finite_diff"))
```

Comment:
Do we need to mark this test as heavy (as the pennylane repo does), or can the runtime be absorbed with little overhead?

Reply:
The test is super lightweight. It takes 1-2 seconds to run all tests.

Reply:
Sorry, correction It takes 0.29 seconds to run all tests.